content/templates.xql

```diff
@@ -596,6 +596,13 @@ declare function templates:form-control($node as node(), $model as map(*)) as no
     let $control := local-name($node)
     return
         switch ($control)
+        case "form" return
+            element { node-name($node) }{
+                $node/@* except $node/@action,
+                attribute action { request:get-parameter('origin', '') },
```

Comment:
Could you replace `request:get-parameter()` here with the templating module's standard method for retrieving parameters?

```xquery
templates:get-configuration($model, "templates:form-control")($templates:CONFIG_PARAM_RESOLVER)("origin")
```

I would expect this to work, but would appreciate it if you could also confirm that it works for you.

Reply:
ok, done, worked here ;-)